axon-server-connector/src/test/java/org/axonframework/axonserver/connector/processor/grpc/FakeEventTrackerStatus.java

```diff
@@ -70,4 +73,15 @@ public boolean isErrorState() {
     public Throwable getError() {
         return exception;
     }
+
+    @Override
+    public OptionalLong getReplayPosition() {
```

Comment:
Based on the discussions we've had, I think this should be replaced for a `getPosition()`.

Reply:
the exposed method name is `getCurrentPosition()`

Reply:
Aah, that makes sense naming wise! :-)